src/components/shared/modals/ConfirmDeployProjectModal.tsx

```diff
@@ -0,0 +1,324 @@
+import { t, Trans } from '@lingui/macro'
+import { Col, Modal, Row, Space, Statistic } from 'antd'
+import { Gutter } from 'antd/lib/grid/row'
+
+import CurrencySymbol from 'components/shared/CurrencySymbol'
+import PayoutModsList from 'components/shared/PayoutModsList'
+import ProjectLogo from 'components/shared/ProjectLogo'
+import TicketModsList from 'components/shared/TicketModsList'
+
+import {
+  useAppSelector,
+  useEditingFundingCycleSelector,
+} from 'hooks/AppSelector'
+import useMobile from 'hooks/Mobile'
+
+import { CurrencyOption } from 'models/currency-option'
+import {
+  formattedNum,
+  formatWad,
+  fromPerbicent,
+  fromPermille,
+} from 'utils/formatNumber'
+import {
+  hasFundingDuration,
+  hasFundingTarget,
+  isRecurring,
+} from 'utils/fundingCycle'
+import { amountSubFee } from 'utils/math'
+import { orEmpty } from 'utils/orEmpty'
+
+import { BigNumber } from 'ethers'
+
+import { useContext } from 'react'
+import { NetworkContext } from 'contexts/networkContext'
+
+import { getBallotStrategyByAddress } from 'constants/ballotStrategies/getBallotStrategiesByAddress'
+
+export default function ConfirmDeployProjectModal({
```

Comment:
any reason for passing this as an arg instead of getting the terminal info from the ProjectContext? i don't think there's any significant tradeoffs, other than the latter may be more consistent

Reply:
Good question! If we were to use context here, we'd have to use `V1ProjectContext`, thus making it v1-specific. In this case I thought `terminalFee` wasn't enough to justify 2 version-specific components.

Reply:
ohh i see, makes sense. i wonder if we should try pulling out some of the `ProjectContext` props that could be shared? we could create a `TerminalContext` that just includes properties from `V1ProjectContext.terminal`.

we may also be able to pull out some things like `projectId`, `createdAt`, `balance`, `overflow`, `isArchived`, etc... but i'm down for us not to commit to any of those things until we need to

Reply:
Yeah more contexts might be an excellent idea 👍 

It turns out that I think we need project-specific modals anyway, discovered this in https://github.com/jbx-protocol/juice-interface/pull/533 😅

Will remove this modal change from this PR 👍